### DISCUSSION TITLE
[Feature Request] Support for Call of Cthulhu

### DISCUSSION THREAD

**Discussion by SsSanzo** [Obsidian-TTRPG-Community/fantasy-statblocks#187] [Feature Requests]
It would be amazing to add support for CALL of CTHULHU by adding automatically calculating and showing half and fifth values, and adding calculated traits (overwritable).

**Comment by lithedreamer** [comment]
If you're still interested, you might find my layout for Delta Green in discussion #41 useful, as they're sibling systems.

**Comment by sigrunixia** [comment]
Marking this as good first issue. While @lithedreamer's layout may fit the bill, this would be a great opportunity for someone to get their feet wet with some CSS.

**Comment by sigrunixia** [comment]
Converting this to a discussion.

**Reply by sigrunixia** [reply]
Conversion Complete!

**Comment by TretinV3** [comment]
Hello, I would like to work on this feature. Should I create a new discussion to talk about how I implement it?

**Reply by valentine195** [reply]
Sure. Might be easier to keep things organized :)
